src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/edit/main.js

```diff
@@ -31,7 +31,10 @@ define(function () {
                             }
                         }
                     },
-                    languageChanger: true
+                    languageChanger: {
+                        url: '/admin/api/languages',
```

Comment:
Didn't you say this is not working anymore, respectively that you want to use the variable you have introduced?